atd-vzv/public/index.html

```diff
@@ -49,7 +49,7 @@
       content="https://visionzero.austin.gov/viewer/summary_img.png"
     />
     <!-- DO NOT ALTER "BELOW" THIS LINE FOR AUTOMATED ENVIRONMENT SNIPPETS -->
-    <!--VZV_HEADER_SNIPPETS-->
+<!-- VZV STAGING SNIPPETS -->
```

Comment:
Why did this update? Given the all caps warning comments, I'm hesitant to merge before reverting this code back